jetty-util/src/main/java/org/eclipse/jetty/util/thread/QueuedThreadPool.java

```diff
@@ -536,6 +537,37 @@ public void setVirtualThreadsExecutor(Executor executor)
         }
     }
 
+    /**
+     * <p>Returns the maximum number of idle threads that are exited for every idle timeout
+     * period, thus shrinking this thread pool towards its {@link #getMinThreads() minimum
+     * number of threads}.
+     * The default value is {@code 1}.</p>
+     * <p>For example, consider a thread pool with {@code minThread=2}, {@code maxThread=20},
+     * {@code idleTimeout=5000} and {@code maxShrinkCount=3}.
+     * Let's assume all 20 threads are executing a task, and they all finish their own tasks
+     * at the same time and no more tasks are submitted; then, all 20 will wait for an idle
+     * timeout, after which 3 threads will be exited, while the other 17 will wait another
+     * idle timeout; then another 3 threads will be exited, and so on until {@code minThreads=2}
+     * will be reached.</p>
+     *
+     * @param shrinkCount the maximum number of idle threads to exit in one idle timeout period
+     */
+    public void setMaxShrinkCount(int shrinkCount)
```

Comment:
I think I like better "evict", because we evict threads and as a consequence the pool shrinks, so `maxEvictCount` here and in other places.